src/core/nsfs.js

```diff
@@ -99,6 +99,8 @@ async function main(argv = minimist(process.argv.slice(2))) {
         await endpoint.start_endpoint({
             http_port,
             https_port,
+            https_port_sts: -1,
```

Comment:
I completely lack the context here so just trying to understand kindly please bear with me...

Are we trying to disable https server and metrics server here? Because in `endpoint.js`, environmental variables won't override these options essentially disabling these 2.

Reply:
that's right!